### DISCUSSION TITLE
RTL

### DISCUSSION THREAD

**Discussion by QusaiImayyah** [QuestPDF/QuestPDF#376] [Q&A] [answered]
When the new version?
You need Right To Left the page and table .....

**Comment by MarcinZiabek** [comment]
Around December. You can track the progress here: https://github.com/QuestPDF/QuestPDF/pull/349

**Reply by QusaiImayyah** [reply]
Ok,We want something alternative for that time.

**Reply by QusaiImayyah** [reply]
![image](https://user-images.githubusercontent.com/20271178/195839472-a54b55b2-4934-44a0-909c-94158802e483.png)
Error Arabic Language......
**The text is correct:**
![image](https://user-images.githubusercontent.com/20271178/195839612-cf7ac98d-45a1-4368-a58f-04f0e6a78035.png)


**What is the solution to this problem at the moment?**

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Full RTL (right-to-left) support, including advanced text shaping for Arabic and other RTL languages, has been available since version 2022.12. You can find the documentation here:
https://www.questpdf.com/api-reference/content-direction.html

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
